docs/reference/commandline/network_create.md

```diff
@@ -148,30 +148,30 @@ fails and Docker Engine returns an error.
-| `com.docker.network.bridge.name`                 | -           | Bridge name to be used when creating the Linux bridge |
-| `com.docker.network.bridge.enable_ip_masquerade` | `--ip-masq` | Enable IP masquerading                                |
-| `com.docker.network.bridge.enable_icc`           | `--icc`     | Enable or Disable Inter Container Connectivity        |
-| `com.docker.network.bridge.host_binding_ipv4`    | `--ip`      | Default IP when binding container ports               |
-| `com.docker.network.driver.mtu`                  | `--mtu`     | Set the containers network MTU                        |
-| `com.docker.network.container_iface_prefix`      | -           | Set a custom prefix for container interfaces          |
+When creating a custom `bridge` network, the following additional options can
+be passed. Some of these have equivalent flags that can be used on the dockerd
+command line or in `daemon.json` to configure the default bridge, `docker0`:
+
+| Network create option                            | Daemon option for `docker0` | Description                                           |
+|--------------------------------------------------|-----------------------------|-------------------------------------------------------|
+| `com.docker.network.bridge.name`                 | -                           | Bridge name to be used when creating the Linux bridge |
+| `com.docker.network.bridge.enable_ip_masquerade` | `--ip-masq`                 | Enable IP masquerading                                |
+| `com.docker.network.bridge.enable_icc`           | `--icc`                     | Enable or Disable Inter Container Connectivity        |
+| `com.docker.network.bridge.host_binding_ipv4`    | `--ip`                      | Default IP when binding container ports               |
+| `com.docker.network.driver.mtu`                  | `--mtu`                     | Set the containers network MTU                        |
+| `com.docker.network.container_iface_prefix`      | -                           | Set a custom prefix for container interfaces          |
 
 The following arguments can be passed to `docker network create` for any
 network driver, again with their approximate equivalents to Docker daemon
 flags used for the docker0 bridge:
 
-| Argument     | Equivalent                        | Description                                |
-|--------------|-----------------------------------|--------------------------------------------|
-| `--gateway`  | -                                 | IPv4 or IPv6 Gateway for the master subnet |
-| `--ip-range` | `--fixed-cidr`, `--fixed-cidr-v6` | Allocate IPs from a range                  |
-| `--internal` | -                                 | Restrict external access to the network    |
-| `--ipv6`     | `--ipv6`                          | Enable or disable IPv6 networking          |
```

Comment:
Did you remove this line by mistake, or on purpose? The daemon has an `--ipv6` flag that controls whether the default bridge is IPv6-capable, so we should keep this line.

Reply:
Oh, thank you ... by mistake while juggling commits (it came back in the next commit). Should be sorted now.